requirements.txt

```diff
@@ -34,8 +34,8 @@ sqlalchemy-migrate==0.7.2 \
 #    --hash=sha256:7f55c768ee26f143fedb11b365a4c57c3bbdb5211e049c7f04b8ae8107c28333
 tempita==0.5.1 \
 #    --hash=sha256:0ebe6938ca7401db79bac279849fffcb5752029150bcb6f3c3edbe7aa9a077d8
-uwsgi==2.0.11.2 \
-#    --hash=sha256:0b889b0b4d2dd3f6625df28cb0b86ec44a68d074ede2d0dfad0b91e88914885c
+uwsgi==2.0.13.1 \
```

Comment:
In the future it's probably best to put things like this in their own PR, but it's not worth it to pull it out.